app/javascript/chat/ChatChannelSettings/ChatChannelSettings.jsx

```diff
@@ -18,6 +18,7 @@ import ChatChannelSettingsSection from './ChatChannelSettingsSection';
 
 export default class ChatChannelSettings extends Component {
   static propTypes = {
+    modFaqEmail: PropTypes.string.isRequired,
```

Comment:
There is prop drill down now. It should be fine since the prop is string and it is 3 levels deep (also adding context might add unwanted complexity).

Reply:
It is being prop drilled, but we have some future work that will probably take care of that. See #9799. For now it's fine that we're [prop drilling](https://kentcdodds.com/blog/prop-drilling).

Reply:
Awesome